test/bats/helpers.bash

```diff
@@ -0,0 +1,129 @@
+REPO_ROOT=$(git rev-parse --show-toplevel)
+COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME:-${REPO_ROOT##*/}}"
+
+GALOY_ENDPOINT=localhost:4002
+SERVER_PID_FILE=$REPO_ROOT/test/bats/.galoy_server_pid
+TRIGGER_PID_FILE=$REPO_ROOT/test/bats/.galoy_trigger_pid
+
+ALICE_PHONE="+16505554328"
+ALICE_CODE="321321"
+
+BOB_PHONE="+198765432113"
+BOB_CODE="321321"
+
+bitcoind_init() {
+  bitcoin_cli createwallet "outside" || true
+  bitcoin_cli -generate 200
+
+  bitcoin_signer_cli createwallet "dev" || true
+  bitcoin_signer_cli -rpcwallet=dev importdescriptors "$(cat ${REPO_ROOT}/test/bats/bitcoind_signer_descriptors.json)"
+}
+
+bitcoin_cli() {
+  docker exec "${COMPOSE_PROJECT_NAME}-bitcoind-1" bitcoin-cli $@
+}
+
+bitcoin_signer_cli() {
+  docker exec "${COMPOSE_PROJECT_NAME}-bitcoind-signer-1" bitcoin-cli $@
+}
+
+start_server() {
+  pid="$(background node lib/servers/graphql-main-server.js)"
+  echo $pid > $SERVER_PID_FILE
+  sleep 5
+}
+
+start_trigger() {
+  pid="$(background node lib/servers/trigger.js)"
+  echo $pid > $TRIGGER_PID_FILE
+  sleep 5
+}
+
+
+stop_server() {
+  [ -f $SERVER_PID_FILE ] && kill -9 $(cat $SERVER_PID_FILE) > /dev/null || true
+}
+
+stop_trigger() {
+  [ -f $TRIGGER_PID_FILE ] && kill -9 $(cat $TRIGGER_PID_FILE) > /dev/null || true
+}
+
+cache_value() {
+  echo $2 > ${BATS_TMPDIR}/$1
+}
+
+read_value() {
+  cat ${BATS_TMPDIR}/$1
+}
+
+gql_query() {
+  cat "${BATS_TEST_DIRNAME:-${REPO_ROOT}/test/bats}/gql/$1.gql" | tr '\n' ' ' | sed 's/"/\\"/g'
+}
+
+exec_graphql() {
+  token_name=$1
+  query_name=$2
+  variables=${3:-"{}"}
+  echo "GQL query -  user: ${token_name} -  query: ${query_name} -  vars: ${variables}"
+  echo  "{\"query\": \"$(gql_query $query_name)\", \"variables\": $variables}"
+
+  if [[ ${token_name} == "anon" ]]; then
+       AUTH_HEADER=""
+  else
+       AUTH_HEADER="Authorization: Bearer $(read_value ${token_name})"
+  fi
+
+  if [[ "${BATS_TEST_DIRNAME}" != "" ]]; then
+    run_cmd="run"
+  else
+    run_cmd=""
+  fi
+
+  ${run_cmd} curl -s \
+       -X POST \
+       ${AUTH_HEADER:+ -H "$AUTH_HEADER"} \
+       -H "Content-Type: application/json" \
+       -d "{\"query\": \"$(gql_query $query_name)\", \"variables\": $variables}" \
+       ${GALOY_ENDPOINT}/graphql
+
+  echo "GQL output: '$output'"
+}
+
+graphql_output() {
+  echo $output | jq -r "$@"
+}
+
+# Run the given command in the background. Useful for starting a
+# node and then moving on with commands that exercise it for the
+# test.
+#
+# Ensures that BATS' handling of file handles is taken into account;
+# see
+# https://github.com/bats-core/bats-core#printing-to-the-terminal
+# https://github.com/sstephenson/bats/issues/80#issuecomment-174101686
+# for details.
+background() {
+  "$@" > /dev/null 2>&1 &
```

Comment:
You should not change this code - you removed the option to be able to get the output of the background tasks.